exploration/open-close-expressions.md

```diff
@@ -220,3 +221,51 @@ Rendered as React this could become:
 _What other solutions are available?_
 _How do they compare against the requirements?_
 _What other properties they have?_
+
+### HTML-like syntax
+
+The goal of this solution is to avoid adding new sigils to the syntax.
+Instead, it leverages the familiarity of the `foo`...`/foo` idiom,
+inspired by HTML and BBCode.
+
+This solution consists of adding new placeholder syntax:
+`{foo}`, `{/foo}`, and `{foo/}`.
+
+```
+This is {html:strong}bold{/html:strong} and this is {html:img alt=|an image|/}.
```

Comment:
If we required the namespace prefix for spannables, that eliminates the problem with unquoted literals (at the cost of some lookahead when parsing). But it would make for some confusion in the regular syntax. Don't forget that we allow `{:function no=operand}` and this looks like an open (is it a syntax error to omit the `/` at the end?)